app/models/Offices.scala

```diff
@@ -16,15 +16,7 @@
 
 package models
 
-import org.scalatest.freespec.AnyFreeSpec
-import org.scalatest.matchers.must.Matchers
-import org.scalatestplus.mockito.MockitoSugar
+abstract class Office(value: String)
```

Comment:
Weird how git thinks that SpecialMentionGuaranteeDetailsSpec.scala 
 has been moved to app/models/Offices.scala